proxy/InkAPI.cc

```diff
@@ -8415,6 +8418,9 @@ TSHttpTxnConfigFind(const char *name, int length, TSOverridableConfigKey *conf,
         cnf = TS_CONFIG_HTTP_RESPONSE_SERVER_STR;
         typ = TS_RECORDDATATYPE_STRING;
       }
+      else if (!strncmp(name, "proxy.config.ssl.client.verify.server", length)) {
```

Comment:
I'd have to see this in context, but how did this else end up on its own line? Shouldn't it be part of the previous }?